src/DotNetLightning.Core/Crypto/Keys.fs

```diff
@@ -248,6 +248,13 @@ and ChannelPubKeys = {
     HtlcBasepoint: HtlcBasepoint
 }
 
+and CommitmentPubKeys = {
```

Comment:
no need for `and`, it's a new type and you're not using it from any type ahead (upper than) of it

Reply:
Fixed.

Reply:
this still has `and`, so not fixed actually